package_io/input_parser.star

```diff
@@ -1,16 +1,16 @@
 DEFAULT_EL_IMAGES = {
-	"geth":			"ethereum/client-go:latest",
+	"geth":			"ethpandaops/geth:4844-devnet-6-e03b5ad",
```

Comment:
I wouldn't change the defaults to 4844 images yet, atleast not until the all-EIP images come out and are somewhat stable

Reply:
reverted to using latest images. Totally forgot about changing these during my tests.